packages/aa/src/index.js

```diff
@@ -83,12 +102,24 @@ function wrappedResolveSync(resolve, depName, packageDir) {
   const depRelativePackageJsonPath = path.join(depName, 'package.json')
   try {
     return resolve.sync(depRelativePackageJsonPath, { basedir: packageDir })
-  } catch (err) {
-    if (!(/** @type {Error} */ (err).message.includes('Cannot find module'))) {
-      throw err
+  } catch (e) {
+    const err = /** @type {Error} */ (e)
+    if (
+      err.message?.includes('Cannot find module') &&
+      resolve !== performantResolve
```

Comment:
I'm pretty sure I don't want this feature. 

If someone's supplying a specific resolver, they should be limited to that resolver's knowledge. Us finding the module would likely be invalid anyway.